user.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * Class User.
+ *
+ * @var string @fname private.
+ * Stores first Name of user.
+ *
+ * @var string @lname private.
+ * Stores Last Name of user.
+ */
```

Comment:
1. `Class User` is not a good comment title, it should be more described.
2. We should add a space at the start of the parameter description.
3. We do not need to put access modifier.